### PR TITLE
Master mobile m2o panel msh final adr

### PR DESCRIPTION
**Pull request by adr-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/views/res_company_views.xml [hunk 1]
```diff
@@ -88,6 +88,16 @@
                                         <field name="phone"/>
                                     </div>
                                 </div>
+                                <div t-else="">
```
Comment (pparidans) [inline odoo/addons/base/views/res_company_views.xml]: event the original looks broken... for 1-2 years

`t-if="record.phone.value"` ... `<field name="email"/>` 🤔

File: addons/web/static/src/js/views/view_dialogs.js [hunk 2]
```diff
@@ -360,29 +353,58 @@ var SelectCreateDialog = ViewDialog.extend({
...
+        if (viewType === 'list') { // add listview specific options
+            _.extend(viewOptions, {
+                hasSelectors: !this.options.disable_multiple_selection,
+                readonly: true,
+
+            }, this.options.list_view_options);
+        }
+        if (viewType === 'kanban') {
+            _.extend(viewOptions, {
+                noDefaultGroupby: true,
```
Comment (pparidans) [inline addons/web/static/src/js/views/view_dialogs.js]: Probably not useful...

File: addons/analytic/views/analytic_account_views.xml [hunk 3]
```diff
@@ -205,14 +205,14 @@
...
+                                <div t-attf-class="#{!selection_mode ? 'text-center' : ''}">
                                    <strong><span><field name="display_name"/></span></strong>
                                 </div>
                                 <hr class="mt8 mb8"/>
                                 <div class="row">
-                                    <div class="col-12 text-center">
+                                    <div t-attf-class="col-12 #{!selection_mode ? 'text-center' : ''}">
                                         <span>
-                                            <strong>Balance: <field name="balance" widget="monetary"/></strong>
+                                            Balance: <field name="balance" widget="monetary"/>
```
[the remark this answers is not in the document]
Comment (msh-odoo) [reply addons/analytic/views/analytic_account_views.xml]: It has been asked in review by EST

File: addons/web/static/src/js/views/view_dialogs.js (hunk 2 above)
Comment (msh-odoo) [reply addons/web/static/src/js/views/view_dialogs.js]: @pparidans It is useful in Selection kanban, we do not want to open grouped kanban view in selection mode

**Pull request closed without merging**